### PR TITLE
Pure refactoring - use nowdoc instead of heredoc

### PR DESCRIPTION
**Pull request by mvorisek**

### CODE REVIEW

**Comment by mvorisek** [comment]
not supported by PHP 7.2

File: src/Selenium2Driver.php [hunk 1]
```diff
@@ -585,44 +585,44 @@ public function getValue($xpath)
...
+                for (var i = 0; i < node.options.length; i++) {
+                    if (node.options[i].selected) {
+                        value.push(node.options[i].value);
+                    }
+                }
 
-return value;
-JS;
+                return value;
+                JS;
```
Comment (stof) [inline src/Selenium2Driver.php:625]: this is a no-go. The driver supports PHP 7.2.

**Comment by stof** [comment]
the issue is not about nowdoc vs heredoc, but about the indented variant of them.

**Comment by mvorisek** [comment]
> the issue is not about nowdoc vs heredoc, but about the indented variant of them.

I find nowdoc safer as no variables interpolation is possibles, but the changed LOC is low when not reindent, thus I will submit it as a part of another PR

**Pull request closed without merging**